README.md

```diff
@@ -84,68 +85,66 @@ You can find more example Policy Packs in the [examples repo](https://github.com
+1. Run `npm install` in the Policy Pack directory.
 
-If you don’t have a stack readily available, you can create a new stack for testing by running `pulumi new aws-typescript` in an empty directory. This AWS example will create an S3 bucket, which is perfect for testing out the template Policy Pack.
+1. Use the `--policy-pack` flag with `pulumi preview` or `pulumi up` to specify the path to the directory containing your Policy Pack when previewing/updating a Pulumi project.
 
-In the stack directory run:
+    If you don’t have a Pulumi project readily available, you can create a new project for testing by running `pulumi new aws-typescript` in an empty directory. This AWS example will create an S3 bucket, which is perfect for testing our Policy.
 
-```sh
-# The `--policy-pack` flag is currently behind the `DEBUG` flag.
-$ PULUMI_DEBUG_COMMANDS=true pulumi preview \
---policy-pack <path-to-new-policy-pack-directory>
-```
+    In the Pulumi project's directory run:
 
-If the stack is in compliance, we expect the output to simply tell us which Policy Packs were run.
+    ```sh
+    PULUMI_DEBUG_COMMANDS=true pulumi preview --policy-pack <path-to-policy-pack-directory>
+    ```
 
-```sh
-$ PULUMI_DEBUG_COMMANDS=true pulumi preview \
---policy-pack ~/policy-pack-typescript
-Previewing update (dev):
+    If the stack is in compliance, we expect the output to simply tell us which Policy Packs were run.
 
-     Type                 Name          Plan
- +   pulumi:pulumi:Stack  test-dev  	create
- +   └─ aws:s3:Bucket     my-bucket     create
+    ```sh
+    PULUMI_DEBUG_COMMANDS=true pulumi preview --policy-pack policy-pack-typescript
+    Previewing update (dev):
 
-Resources:
-    + 2 to create
+        Type                 Name          Plan
+    +   pulumi:pulumi:Stack  test-dev  	create
+    +   └─ aws:s3:Bucket     my-bucket     create
 
-Permalink:
-...
-```
+    Resources:
+        + 2 to create
 
-We can then edit the stack code in `index.ts` to specify the ACL to be public-read.
+    Permalink:
+    ...
+    ```
 
-```typescript
-const bucket = new aws.s3.Bucket("my-bucket", {
-    acl: "public-read",
-});
-```
+1. We can then edit the stack code to specify the ACL to be public-read.
 
-We then run the pulumi preview command again and this time get an error message indicating we failed the preview because of a policy violation.
+    ```typescript
+    const bucket = new aws.s3.Bucket("my-bucket", {
+        acl: "public-read",
+    });
+    ```
 
-```sh
-$ PULUMI_DEBUG_COMMANDS=true pulumi preview \
---policy-pack ~/policy-pack-typescript
-Previewing update (dev):
+1. We then run the `pulumi preview` command again and this time get an error message indicating we failed the preview because of a policy violation.
 
-     Type                 Name          Plan       Info
- +   pulumi:pulumi:Stack  test-dev  	create     1 error
- +   └─ aws:s3:Bucket     my-bucket     create     1 error
+    ```sh
```

Comment:
Do the numbers still get generate correctly? I found when I moved these over before the number would restart.

Reply:
Good catch. Without indenting, the numbers always stay "1". Fixed.